packages/api/src/platforms/vtex/resolvers/session.ts

```diff
@@ -0,0 +1,19 @@
+import type { Context } from '..'
+import type { MutationSessionArgs } from '../../../__generated__/schema'
+import ChannelParser from '../utils/channel'
+
+export default async function sessionResolvers(
```

Comment:
nit: validateCart uses `export const validateCart` instead of `export default`.

Reply:
What's the impact of this change? Can you explain the motivation?